### PR TITLE
Auto session storage key

### PR DESCRIPTION
**Pull request by VanoDevium**
#1766

### CODE REVIEW

File: web/auth.go [hunk 1]
```diff
@@ -115,3 +117,8 @@ func (h *handler) clearSession(w http.ResponseWriter, r *http.Request) {
 
 	http.Redirect(w, r, h.config.Base, http.StatusTemporaryRedirect)
 }
+
+func getSessionStorageKey(username string, password string) []byte {
```
Comment (amir20) [inline web/auth.go:121]: nit: Call this `generateSessionStorateKey`

**Review by amir20 — CHANGES_REQUESTED** [review]
Nicely done. Just one quick change. I can merge after.

File: web/auth.go (hunk 1 above)
Comment (VanoDevium) [reply web/auth.go]: done :)

Comment (VanoDevium) [reply web/auth.go]: dmn, you misstype))

**Pull request merged**